File: Assignment9/Program5.c
// 5. Accept number from user and display below pattern.
// Input :  8
// Output : 2 4 6 8 10 12 14 16

#include <stdio.h>
void Pattern(int iNo)
{
    int iCnt = 0;
    int iMul = 1;
    for(iCnt = 1; iCnt <= iNo;iCnt++)
    {
        iMul = iCnt * 2;
        printf("%d ",iMul);
    }
}
int main()
{
    int iValue = 0;
    printf("Enter number of elements");
    scanf("%d", &iValue);
    Pattern(iValue);
    return 0;
}

File: Assignment32/Program5.java
// 5. Write Java program which accept N numbers from user and display all such elements
//  which are multiples of 11. 

// Input :  N :   6 
//   Elements : 85 66 3 55 93 88  
// Output : 66 55 88  

import java.util.*;

class Program5
{
	public static void main(String[] args) 
	{
		Scanner sobj = new Scanner(System.in);
		int iSize = 0 , iCnt = 0;

		System.out.println("Enter number of Elements");
		iSize = sobj.nextInt();

		int Arr[] = new int[iSize];

		System.out.println("Enter Elements");
		for(iCnt = 0;iCnt < Arr.length;iCnt++)
		{
			Arr[iCnt] = sobj.nextInt();
		}

		ArrayDemo adobj = new ArrayDemo();
		adobj.Display(Arr);
	}
}

class ArrayDemo                                            
{  
    public void Display(int Arr[])  
    {   
    	int iCnt = 0;

    	for(iCnt = 0;iCnt < Arr.length;iCnt++)
    	{
    		if(Arr[iCnt] % 11 == 0)
    		{
    			System.out.println("Multiples of 11 :"+Arr[iCnt]);
    		}
    	}
    }      
} 